README.md

```diff
@@ -59,3 +61,5 @@ Bloxstrap also only runs whenever necessary, so it doesn't stay running in the b
 * taskmanager ([@Mantaraix](https://github.com/Mantaraix)) - Helping with designing the new menu look and layout.
 * [@Extravi](https://github.com/Extravi) - Allowing their presets to be bundled with Bloxstrap, and helping with improving UX.
 * [@axstin](https://github.com/axstin) - Making [rbxfpsunlocker](https://github.com/axstin/rbxfpsunlocker), which was used for Bloxstrap's FPS unlocking up until v2.2.0.
+
+[^1]: This happens because it's an unknown program, not because it's actually detected as being malicious.
```

Comment:
Should this really be a footnote? I feel like this is a pretty important detail to outline.

Reply:
i thought it was explanation, that's why footnote
if you want so, i can definitely change that